src/ValuedEnum.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Saritasa;
+
+/**
+ * The base class for enums that associated with one string value.
```

Comment:
Shouldn't it be '... with one numeric value'?

Reply:
Yes, it's typo